### PR TITLE
Nbsp removal

### PR DESCRIPTION
**Pull request by anjali-dhanuka**
# Description
Include a summary of the change and relevant motivation/context. List any dependencies that are required for this change.

Fixes # [ISSUE]

# Type of Change:
**Delete irrelevant options.**

- Code
- Quality Assurance
- User Interface
- Outreach
- Documentation

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

# How Has This Been Tested?
Describe the tests you ran to verify your changes. Provide instructions or GIFs so we can reproduce. List any relevant details for your test.

# Checklist:
**Delete irrelevant options.**

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged 

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Comment by Monal5031** [comment]
@anjali-dhanuka This will cause some tests to fail, because AFAIR there are some tests which use these blank spaces for separating the data using `split()` method.

**Comment by anjali-dhanuka** [comment]
I'll fix them locally and push that. @Monal5031

File: vms/volunteer/tests/test_volunteerReport.py [hunk 1]
```diff
@@ -65,7 +65,7 @@ def verify_shift_details(self, hours):
         :param total_shifts: Total number of shifts as filled in form.
         :param hours: Total number of hours as filled in form.
         """
-        total_no_of_hours = self.report_page.get_shift_summary().split(' ')[-1].strip('\n')
+        total_no_of_hours = self.report_page.get_report_hours()
```
Comment (Monal5031) [inline vms/volunteer/tests/test_volunteerReport.py]: remove that previous method from pom object too, otherwise we'd have unused code.

**Comment by anjali-dhanuka** [comment]
@Monal5031 Done please review

**Comment by anjali-dhanuka** [comment]
Hi @abhishekarya286 @Monal5031 @kamsuri @nida Please review

**Comment by Monal5031** [comment]
@anjali-dhanuka complete the changes by @kamsuri so this can be merged asap.

**Comment by anjali-dhanuka** [comment]
done @kamsuri

**Pull request merged**